tinygrad/codegen/uops.py

```diff
@@ -60,10 +60,12 @@ def ge(self, x): return -self.lt(x)
   def max(self, x): return UOp.alu(BinaryOps.MAX, self, x)
   def min(self, x): return -UOp.alu(BinaryOps.MAX, -self, -x)
   @staticmethod
+  @functools.lru_cache(None)
```

Comment:
These two caches give a big speedup.

Reply:
Would merge a PR with just this one.

Reply:
https://github.com/tinygrad/tinygrad/pull/5156